_posts/2022-11-17-Multiple-Authentication.md

```diff
@@ -0,0 +1,51 @@
+
+We would love to hear any additional use-cases you have that this feature can meet!
+
+## How can you use this feature?
+
+OpenSearch Dashboards admins can enable single or multiple authentication options on demand and customize the integrated Login UI by editing opensearch_dashboards.yml. For the essential settings required to setup multiple authentication, see [**Multiple Authentication Options**]({{site.url}}{{site.baseurl}}/security-plugin/configuration/multi-auth/). Once these options are configured, the Login UI will be automatically updated. For example, the login UI below shows both basic authentication and SSO enabled:
+
+![Default Login UI](/assets/media/blog-images/2022-11-14-multiple-authentication/default-login-ui.png){: .img-fluid}
+
+## How can you use this feature?
+
+When Dashboards bootstraps and loads security settings from `opensearch-dashboards.yml`, the Dashboards client can tell whether Single-Authentication Mode or Multiple-Authentication Mode is enabled by evaluating the Multiple Authentication feature flag and authentication type setting.
+
+* If **Single-Authentication Mode** is detected, only authentication handler for corresponding authentication type defined by *`attribute:opensearch_security.auth.type`* will be registered with the client. All the login requests flow to the dedicated IDP to complete identification and authentication.
+* If **Multiple-Authentication Mode** is detected, OSD will evaluate authentication handlers required for each individual authentication type defined by *`attribute:opensearch_security.auth.type`* and form a compound authentication handler, which will be registered with OSD client instead. This compound authentication handler is named as multi-auth handler. Multi-auth handler is able to redirect the authentication request to various IDP endpoints for authentication interchangeably based on the authentication type defined within Login request. 
+
+**High Level Diagram:**
+![High Level Diagram](/assets/media/blog-images/2022-11-14-multiple-authentication/high-level-diagram.png){: .img-fluid}
+
+## Getting started
+
+For more information on setting up and exploring this feature, see the OpenSearch [documentation]({{site.url}}{{site.baseurl}}/security-plugin/configuration/multi-auth/). To leave feedback, visit the [OpenSearch Forum](https://forum.opensearch.org/t/feedback-experimental-feature-connect-to-external-data-sources/11144). If you're interested in contributing to this feature, consider contributing to the OpenSearch Dashboards [repository](https://github.com/opensearch-project/security-dashboards-plugin).
```

Comment:
Can we use a word other than "contributing" for one of the instances in order to avoid repetition?

Reply:
Change it to `If you're interested in contributing to this feature, please check the OpenSearch Dashboards [repository](https://github.com/opensearch-project/security-dashboards-plugin) for more details.`